### PR TITLE
DEVX-2281: Beginner Cloud CLI tutorial improvements: fix documentatio…

### PR DESCRIPTION
**Pull request by ybyzek**
…n issues

### Description 

https://confluentinc.atlassian.net/browse/DEVX-2281
https://github.com/confluentinc/examples/issues/869

_What behavior does this PR change, and why?_

Fixes bugs in the tutorial

### Author Validation

_Describe the validation already done, or needs to be done, by the PR submitter._

- [x] Documentation

- [x] ccloud/beginner-cloud

### Reviewer Tasks

_Describe the tasks/validation that the PR submitter is requesting to be done by the reviewer._

I have validated a local docs build and compared to tutorial run, so looking just for visual inspection

### CODE REVIEW

File: ccloud/docs/beginner-cloud.rst [hunk 1]
```diff
@@ -233,29 +231,23 @@ Produce and consume records with Confluent Cloud CLI
...
+#. Start producing to this topic ``demo-topic-1`` by running the following command:
 
    .. code-block:: bash
 
-         (for i in `seq 1 10`; do echo "${i}" ; done) | \
-           ccloud kafka topic produce demo-topic-1
+      ccloud kafka topic produce demo-topic-1
 
-#. Verify your output resembles:
+#. The CLI waits for you to type data at the prompt, so type a few characters each on a new line. For example, type th enumbers 1 through 5
```
Comment (awalther28) [inline ccloud/docs/beginner-cloud.rst]:
```suggestion
#. The CLI waits for you to type data at the prompt, so type a few characters each on a new line. For example, type the numbers 1 through 5:
```

Comment (awalther28) [reply ccloud/docs/beginner-cloud.rst]: Just to note, the start.sh script produces numbers 1-10.

File: ccloud/docs/beginner-cloud.rst [hunk 2]
```diff
@@ -264,23 +256,18 @@ Produce and consume records with Confluent Cloud CLI
...
       ccloud kafka topic consume demo-topic-1 -b
 
-#. Verify your output resembles:
+#. Verify your output resembles the following. It is expected to be out of order because of round-robin partitioner:
 
    .. code-block:: text
 
       Starting Kafka Consumer. ^C or ^D to exit
-      2
+      1
```
Comment (awalther28) [inline ccloud/docs/beginner-cloud.rst:264]: Just to note, the start.sh script consumes numbers 1-10.

File: ccloud/docs/beginner-cloud.rst [hunk 3]
```diff
@@ -328,14 +315,15 @@ Create a new service account with an API key/secret pair
...
+   and secret for the service account. Substitute your values for the bootstrap
+   server and credentials just created.
 
    .. code-block:: text
 
        sasl.mechanism=PLAIN
        security.protocol=SASL_SSL
        bootstrap.servers=pkc-4kgmg.us-west-2.aws.confluent.cloud:9092
-       sasl.jaas.config=org.apache.kafka.common.security.plain.PlainLoginModule required username\="ESN5FSNDHOFFSUEV" password\="nzBEyC1k7zfLvVON3vhBMQrNRjJR7pdMc2WLVyyPscBhYHkMwP6VpPVDTqhctamB";
+       sasl.jaas.config=org.apache.kafka.common.security.plain.PlainLoginModule required username='ESN5FSNDHOFFSUEV' password='nzBEyC1k7zfLvVON3vhBMQrNRjJR7pdMc2WLVyyPscBhYHkMwP6VpPVDTqhctamB';
```
Comment (awalther28) [inline ccloud/docs/beginner-cloud.rst:326]: This also differs in the start.sh

**Review by awalther28 — COMMENTED** [review]
Thanks for the updates. I built the docs locally and used the start.sh to validate the output.
I think with a few of these changes should also make it to the start.sh, namely changing the number of ints produced and consumed AND the 'sasl.jaas.config' change (also mentioned in comments in review).

File: ccloud/docs/beginner-cloud.rst (hunk 1 above)
Comment (ybyzek) [reply ccloud/docs/beginner-cloud.rst]: @awalther28 good catch.  Dropping script down to 5 as well.

File: ccloud/docs/beginner-cloud.rst (hunk 2 above)
Comment (ybyzek) [reply ccloud/docs/beginner-cloud.rst:264]: @awalther28 good catch.  Dropping script down to 5 as well.

File: ccloud/docs/beginner-cloud.rst (hunk 3 above)
Comment (ybyzek) [reply ccloud/docs/beginner-cloud.rst:326]: 👍 and resolved

**Comment by ybyzek** [comment]
> Thanks for the updates. I built the docs locally and used the start.sh to validate the output.
> I think with a few of these changes should also make it to the start.sh, namely changing the number of ints produced and consumed AND the 'sasl.jaas.config' change (also mentioned in comments in review).

Thank you for the review @awalther28 .  I believe I have addressed all feedback.

**Review by awalther28 — APPROVED** [review]
@ybyzek looks like you have :) I don't have anything else to add

**Pull request merged**